index.html

```diff
@@ -53,7 +53,12 @@ <h2>Software</h2>
             <td>.gif</td>
             <td>image/gif</td>
           </tr>
-        </tbody>
+            <tr>
+            <td>GZIP-hul</td>
+            <td>Gzip</td>
+            <td>.gz</td>
+            <td>application/gzip</td>
+          </tr>        </tbody>
```

Comment:
Again, it's good to keep unconnected `<tags>` on different lines with consistent indentation for readability.